pyvista/core/filters/data_set.py

```diff
@@ -3791,9 +3792,9 @@ def compute_derivative(dataset, scalars=None, gradient=True,
 
         Examples
         --------
-        First, plot the random hills dataset with the active elevation
-        scalars.  These scalars will be used for the derivative
-        calculations.
+        First, plot the hills example with the active elevation
+        scalars.  These scalars will be used to compute for
+        derivative calculations.
```

Comment:
I'd prefer the original version here (probably what ended up in the dataset filters PR?).

Reply:
Reverted in 55ba3daa9ec943acb388ac83608d7bd7ca5341b1.